### PR TITLE
Add Amendment to rules-cn and rules-en

### PR DESCRIPTION
**Pull request by JackyLHH**

### CODE REVIEW

**Comment by JackyLHH** [comment]
1. 修改了规则中的 Markdown 语法错误（多余符号）；
2. 新增了“补充规则”：Matt 的元规则提案 - [Ban incentivized voting in DAO](https://dao.ckb.community/thread/vot-ban-incentivized-voting-in-dao-43212)

---

1. Modified the Markdown syntax error (extra symbols) in the rules;
2. Added "supplemental rules": Matt's meta-rule proposal - [Ban incentivized voting in DAO](https://dao.ckb.community/thread/vot-ban-incentivized-voting-in-dao-43212)

File: rules-cn.md [hunk 1]
```diff
@@ -98,3 +98,8 @@ DAO 支持两种类型的治理：
     - 在过程中，申请团队有义务向社区同步之后开展工作的进展，比如定期或达成某个 milestone 后，以 **[Status Update]** 为标题开头在论坛给社区汇报工作进展。保持良好的沟通才有利于团队得到社区的帮助，甚至找到志同道合的同伴一起前行。
     - 社区成员也有权利监督申请团队的工作效果和资金使用，并以论坛发帖的方式进行评论或探讨。
 - 如果是修改元规则提案，此类提案通过之后，由于可能会涉及必要的系统开发等因素，无法马上开始执行。对于此类提案，管理委员会应在二周之内给出实施计划，并定期进行状态更新，直到最终实施。
+
+## 补充规则
```
Comment (TabrisCKB) [inline rules-cn.md]: 建议使用 “修正案”

File: rules-en.md [hunk 2]
```diff
@@ -99,4 +99,8 @@ After the proposal is passed, it will enter the Execution Stage.
...
     - Community members also have the right to monitor the effectiveness of the applicant team's work and use of funds and to comment or discuss in the forum.
-- In the case of proposals to modify meta-rules, the implementation of such proposals cannot start immediately after they are adopted because they may involve necessary system development and other factors. For such proposals, the management committee should give an implementation plan within two weeks and make regular status updates until final implementation.
\ No newline at end of file
+- In the case of proposals to modify meta-rules, the implementation of such proposals cannot start immediately after they are adopted because they may involve necessary system development and other factors. For such proposals, the management committee should give an implementation plan within two weeks and make regular status updates until final implementation.
+
+## Supplementary Rules
```
Comment (TabrisCKB) [inline rules-en.md]: Use "Amendment" instead of "Supplementary Rules"?

**Comment by JackyLHH** [comment]
将 “补充规则” 修改为 “修正案”。
Change “supplemental rules” to "Amendment".

**Review by TabrisCKB — APPROVED** [review]

**Pull request merged**